chain/actions/Cargo.toml

```diff
@@ -41,16 +41,16 @@ smart-default = { workspace = true }
 thiserror = { workspace = true }
 
 bindings = { workspace = true }
+chain-types = { workspace = true }
+hopr-async-runtime = { workspace = true }
+hopr-chain-rpc = { workspace = true }
 hopr-crypto-types = { workspace = true }
 hopr-db-sql = { workspace = true }
-chain-rpc = { workspace = true }
-chain-types = { workspace = true }
+hopr-internal-types = { workspace = true }
 hopr-metrics = { workspace = true, optional = true }
 hopr-platform = { workspace = true }
-hopr-async-runtime = { workspace = true }
-tracing = { workspace = true }
-hopr-internal-types = { workspace = true }
 hopr-primitive-types = { workspace = true }
+tracing = { workspace = true }
```

Comment:
Pls move this dependency into external ones way upper. This paragraph is a collection of internal workspace dependencies.

Reply:
Done.